### PR TITLE
adding information about mugc

### PR DESCRIPTION
**Pull request by ismferd**
I have added more information about mugc, in order to help with it.

### CODE REVIEW

File: tools/ops/README.md [hunk 1]
```diff
@@ -11,6 +11,18 @@ The only required argument is `-c`: a list of config (policy) files.
...
 
+The policy file must be like:
+
+```
+policies:
+  - name: delete
+    resource: ebs
+```
+
+**You can't choose which CloudCustodian lambda will be delete**
```
Comment (kapilt) [inline tools/ops/README.md]: you can actually, --prefix or --regex can point to the full name of the lambda

**Comment by kapilt** [comment]
Thanks for the pull request, some comments inline.  If you don't mind could you sign the CLA per the dev docs? Direct link here: https://docs.google.com/forms/d/e/1FAIpQLSfwtl1s6KmpLhCY6CjiY8nFZshDwf_wrmNYx1ahpsNFXXmHKw/viewform

File: tools/ops/README.md (hunk 1 above)
Comment (ismferd) [reply tools/ops/README.md]: Oh great, I will add more info for `--prefix`
I don't see any references to `--regex` in the setup function:
```
def setup_parser():
    parser = argparse.ArgumentParser()
    parser.add_argument("configs", nargs='*', help="Policy configuration file(s)")
    parser.add_argument(
        '-c', '--config', dest="config_files", nargs="*", action='append',
        help="Policy configuration files(s)", default=[])
    parser.add_argument(
        '-r', '--region', action='append', dest='regions', metavar='REGION',
        help="AWS Region to target. Can be used multiple times, also supports `all`")
    parser.add_argument('--dryrun', action="store_true", default=False)
    parser.add_argument(
        "--profile", default=os.environ.get('AWS_PROFILE'),
        help="AWS Account Config File Profile to utilize")
    parser.add_argument(
        "--prefix", default="custodian-",
        help="The Lambda name prefix to use for clean-up")
    parser.add_argument("-p", "--policies", default=None, dest='policy_filter',
                        help="Only use named/matched policies")
    parser.add_argument(
        "--assume", default=None, dest="assume_role",
        help="Role to assume")
    parser.add_argument(
        "-v", dest="verbose", action="store_true", default=False,
        help='toggle verbose logging')
    return parser
```

Comment (kapilt) [reply tools/ops/README.md]: its a recent addition on trunk see policy-regex https://github.com/cloud-custodian/cloud-custodian/blob/master/tools/ops/mugc.py#L171

Comment (ismferd) [reply tools/ops/README.md]: oohhh, I'm out to date, sorry.

**Comment by ismferd** [comment]
@kapilt , I added some info about `--regex-policy` too, take a look, please

**Comment by ismferd** [comment]
Can you take a look @kapilt ?

**Comment by kapilt** [comment]
added some more content wrt to how mugc operates, and the new --present flag (which inverts the behavior), and updated --help output.  looks good.

**Pull request merged**